src/Viren.Execution/Dtos/CalculationResultDto.cs

```diff
@@ -22,6 +18,6 @@ public CalculationResultDto()
         public DateTime TimeStamp { get; set; }
         public int ElapsedMilliseconds { get; set; }
 
-        public IDictionary<string, object> Result { get; set; }
+        public JObject Result { get; set; }
```

Comment:
IDictionary<string, object> gebruiken in de viren.net client ipv. JObject. Dan moeten clients niet meer omzetten van uit JSON.

Reply:
Fixed